cmd/kwild/config/flags.go

```diff
@@ -119,4 +119,6 @@ to instead run a dedicated seeder like https://github.com/kwilteam/cometseed.`))
 	flagSet.MarkDeprecated("app.snapshots.snapshot-dir", "this value is no longer configurable")
 	flagSet.String("chain.statesync.snapshot-dir", "", "Chain state sync snapshot directory")
 	flagSet.MarkDeprecated("chain.statesync.snapshot-dir", "this value is no longer configurable")
+	flagSet.BoolVar(&cfg.AppConfig.Snapshots.DEPRECATED_Enabled, "app.snapshots.enabled", cfg.AppConfig.Snapshots.DEPRECATED_Enabled, "Enable snapshots")
```

Comment:
It looks like you still need to use the DEPRECATED_Enabled config. If a user sets this to true, it should set the `enable` config. If a user sets both, it should error.